src/index.js

```diff
@@ -25,13 +24,12 @@ window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){va
 
   apply(compiler) {
     compiler.hooks.compilation.tap('HtmlWebpackNewRelicPlugin', (compilation) => {
-      compilation.hooks.htmlWebpackPluginBeforeHtmlProcessing.tap(
+      HtmlWebpackPlugin.getHooks(compilation).alterAssetTagGroups.tap(
         'HtmlWebpackNewRelicPlugin',
         (data) => {
-          const $ = cheerio.load(data.html);
-          $('body').append(this.newrelicstring);
-          Object.assign(data, { html: $.html() });
-        },
+          const newRelicScriptTag = HtmlWebpackPlugin.createHtmlTagObject('script', {}, this.newrelicstring);
```

Comment:
Does this createHtmlTagObject need to specify the script type?  `text/javascript`

Reply:
Unsure, but it probably can't hurt. I'll add.